config/log4cxx.properties

```diff
@@ -2,7 +2,7 @@ log4j.rootLogger=INFO, A1, F1
 log4j.appender.A1=ConsoleAppender
 log4j.appender.A1.Target=System.err
 log4j.appender.A1.layout=PatternLayout
-log4j.appender.A1.layout.ConversionPattern=%X{PID} %-5p %c: %m%n
+log4j.appender.A1.layout.ConversionPattern=%X{PID} %-5p %d{yyyy-MM-ddThh:mm:ss.sss} %c: %m%n
```

Comment:
Opened https://github.com/lsst/ctrl_pool/pull/16 

I assume we don't need another ticket?